services/src/integ-test/resources/application-integ-asb.yaml

```diff
@@ -3,10 +3,8 @@ spring:
     servicebus:
       internal:
         targetAppQueue:
-          MCOL: ${MCOL.Queue:civil_sdt_inbound}
-          MCOLS: ${MCOLS.Queue:civil_sdt_inbound}
-          TEST1: ${MCOL.Queue:civil_sdt_inbound}
-      connection-string: ${SDT_QUEUE_CONNECTION_STRING:Endpoint=sb://hmc-servicebus-demo.servicebus.windows.net/;SharedAccessKeyName=Root;SharedAccessKey=wS5bXXPPip8KKEKc9A4/P6krOazMvSCwcdW4PmbsD0A=;EntityPath=civil_sdt_inbound}
+          TEST1: ${CIVIL_SDT_QUEUE:civil_sdt_inbound}
+      connection-string: ${CIVIL_SDT_SERVICEBUS_CONNECTION_STRING:Endpoint=sb://hmc-servicebus-demo.servicebus.windows.net/;SharedAccessKeyName=Root;SharedAccessKey=wS5bXXPPip8KKEKc9A4/P6krOazMvSCwcdW4PmbsD0A=;EntityPath=civil_sdt_inbound}
```

Comment:
We'll need to rotate the HMC Demo Service Bus keys as these should not be exposed publicly.

Reply:
removed this